ffs/integrationtest/integration_test.go

```diff
@@ -989,6 +1007,67 @@ func TestRemove(t *testing.T) {
 	require.Equal(t, api.ErrNotFound, err)
 }
 
+// This isn't very nice way to test for repair. The main problem is that now
+// deal start is buffered for future start for 10000 blocks at the Lotus level.
+// Se we can't wait that much on a devnet. That setup has some ToDo comments so
+// most prob will change and we can do some nicier test here.
+// Better than no test is some test, so this tests that the repair logic gets triggered
+// and the related Job ran successfully.
+func TestRepair(t *testing.T) {
```

Comment:
Read comment above.
Its a quite dense test fo walkaround the problem, if this becomes too flaky might deserve being skipped. For the moment is checking green.